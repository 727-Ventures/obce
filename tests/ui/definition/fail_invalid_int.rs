#[obce::definition(id = 0x12345678)]
pub trait Trait {}

fn main() {}
