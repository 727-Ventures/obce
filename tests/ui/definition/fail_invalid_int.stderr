error: number too large to fit in target type
 --> tests/ui/definition/fail_invalid_int.rs:1:25
  |
1 | #[obce::definition(id = 0x12345678)]
  |                         ^^^^^^^^^^
